cli/ballerina-cli/src/main/java/io/ballerina/cli/utils/JarFileWriter.java

```diff
@@ -15,7 +15,7 @@
  * specific language governing permissions and limitations
  * under the License.
```

Comment:
We do not need the JarFileWriter lets merge this PR and create a new issue to remove this class. I guess we use the writer in some test utils which also should be removed.

Reply:
I have created an issue
https://github.com/ballerina-platform/ballerina-lang/issues/27395